### DISCUSSION TITLE
Incomplete installation

### DISCUSSION THREAD

**Discussion by smallboy69** [Security-Onion-Solutions/securityonion#13303] [2.4] [answered]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

256

### Storage for /

4T

### Storage for /nsm

2T

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

hi Team

i downloaded the image file from here

https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO.md

created a bootable usb and selected a Security Onion with GUI Desktop 

but the application did not install completely as no so commands 

and there are no log files...in /opt/so/log

the GUI is installed but not the application..

appreciate your advise

Thanks

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment, marked as answer]
When you select the Desktop option, it only installs the Desktop environment and nothing else. This is primarily intended for connecting to an existing Security Onion deployment. For more information, please see:
https://docs.securityonion.net/en/2.4/desktop.html

**Reply by smallboy69** [reply]
thank you @dougburks 

this makes sense, appreciate your confirmation

Regards
Joseph

**Reply by smallboy69** [reply]
@dougburks 

hi Sir

just curious as to why opencanary is not integrated in the latest Version of Security Onion...

in terms of profiling threat actors strategic attack vectors, would a honeypot not yield deep insight into the mindset of threat actors

Thanks

**Reply by dougburks** [reply]
Security Onion includes OpenCanary in our Intrusion Detection Honeypot (IDH) node:
https://docs.securityonion.net/en/2.4/idh.html

If you have further questions or problems, please start a new discussion with appropriate title rather than continuing this discussion.

**Reply by smallboy69** [reply]
Thanks @dougburks appreciate the advise 

and noted on the new discussion with appropriate title
